include/zephyr/drivers/clock_control/tisci_clock_control.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2025 Texas Instruments
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_CLOCK_CONTROL_TISCI_CLOCK_CONTROL_H_
+#define ZEPHYR_DRIVERS_CLOCK_CONTROL_TISCI_CLOCK_CONTROL_H_
+#include <stdint.h>
```

Comment:
Space before this line, and one before the final #endif. Just follow what ever other header in this directory are doing.